common/quotas/calculator/cluster_aware_quota_calculator.go

```diff
@@ -22,7 +22,12 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package quotas
+package calculator
```

Comment:
I have to move those to a new package. Otherwise will be a cycle dependency between log and quotas package (throttled logger depends on the quotas package, and I want to emit logs for when computing the quota).